kubernetes/clusters/dev/values.yaml

```diff
@@ -23,7 +23,7 @@ zookeeper:
     failureThreshold: 8
 
 kafka:
-  replicaCount: 1
+  replicaCount: 3
```

Comment:
```suggestion
  replicaCount: 1
```

Reply:
This will help in overriding - https://github.com/hypertrace/raw-spans-grouper/pull/31